py-polars/polars/series/series.py

```diff
@@ -2992,6 +2992,7 @@ def to_physical(self) -> Series:
         - :func:`polars.datatypes.Time` -> :func:`polars.datatypes.Int64`
         - :func:`polars.datatypes.Duration` -> :func:`polars.datatypes.Int64`
         - :func:`polars.datatypes.Categorical` -> :func:`polars.datatypes.UInt32`
+        - ``List(Categorical)`` -> ``List(UInt32)``
```

Comment:
This is too specific. We could just mention that physical types are converted recursively.